styles.css

```diff
@@ -0,0 +1,886 @@
+    margin-left: 1rem;
+  }
+
+  .sub-section-header-white {
+    padding: 0;
+  }
+
+  .align-center {
+    align-items: center;
+  }
+
+  .red-color {
+    color: #ff5438 !important;
+  }
+
+  .program-list h3 {
+    word-break: break-all;
+    width: 30%;
+    padding-left: 0.5rem;
+    white-space: nowrap;
+    align-items: center;
+    text-align: left;
+  }
+
+  .join-btn {
+    justify-content: center;
+    margin: 2rem 0 2rem 3rem;
+  }
+
+  .join-btn p {
+    background-color: #ff4f38;
+    padding: 1.5rem 4rem;
+    text-align: center;
+    font-size: 12px;
+  }
+
+  .join-btn:hover {
+    cursor: pointer;
+  }
+
+  .image-container {
+    width: 125px;
+    height: 125px;
+    position: relative;
+    margin-left: -5rem;
+  }
+
+  .speaker-image {
+    width: 107px;
+    height: 106px;
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    z-index: 1;
+  }
+
+  .see-more {
+    border: 1px solid rgb(165, 161, 161);
+    padding: 0.5rem;
+    border-radius: 5px;
+    font-style: normal;
+    margin-left: -3rem;
+    margin-right: 3rem;
+  }
+
+  .see-more > div {
+    font-size: 13px;
+    color: #2c2c2c;
+    font-weight: 700;
+  }
+
+  .fa {
+    color: red;
+  }
+
+  .contact p {
+    margin-bottom: 1.5rem;
+    text-align: center;
+    font-size: 15px;
+    margin-left: 10rem;
+    font-weight: 700;
+    line-height: 25px;
+    width: 290px;
+    height: 60px;
+    font-style: normal;
+    margin-top: 2rem;
+  }
+
+  .sub-section-white {
+    background-color: white;
+    padding-left: 7rem;
+  }
+
+  .about-main-title {
+    font-size: 30px;
+    font-weight: 800;
+    color: #ff5438;
+    padding: 0;
+    margin-left: 1rem;
+  }
+
+  .about-main-paragraph {
+    line-height: 25px;
+    margin: 1rem 3rem;
+    padding: 0;
+    background-color: white;
+    color: black;
+    border: 2px solid white;
+    margin-left: 3rem;
+  }
+
+  .section-header-black {
+    text-align: center;
+    margin: 1rem 5rem;
+    margin-left: 1rem;
+  }
+
+  .logo-image {
+    margin: 1rem 4rem 0 0;
+    border: 1px solid rgb(230, 223, 223);
+    padding: 0;
+  }
+
+  .mob-text {
+    color: #515151;
+    margin-right: 6rem;
+  }
+
+  .simple-cards > div {
+    margin: 1rem 0 0 -4rem;
+    height: 40vh;
+  }
+
+  nav {
+    padding: 0.5rem 17.5rem;
+    background-color: rgba(255, 255, 255, 0.6);
+    margin-bottom: 6rem;
+    position: fixed;
+    width: 100%;
+    justify-content: space-between;
+    padding-right: 50%;
+    z-index: 2;
+  }
+
+  .about_mobile_nav {
+    padding: 0.5rem 20rem;
+    background-color: rgba(255, 255, 255, 0.6);
+    margin-bottom: 6rem;
+    position: fixed;
+    width: 100%;
+    justify-content: space-between;
+    padding-right: 50%;
+    z-index: 2;
+  }
+
+  .menu-list li {
+    margin: 0 0;
+    display: flex;
+    justify-content: flex-start;
+  }
+
+  .menu {
+    padding: 8rem 0;
+    background-color: #282b32;
+    position: fixed;
+    top: 0;
+    width: 60%;
+    height: 100vh;
+    z-index: 3;
+  }

```

Comment:
- [ ] Kindly increase the `width` of this overlay element to cover the whole screen for good user experience. 👍 

## Your design
<img width="277" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235204319-5bf98cd1-eab2-4c42-8bd3-23645415d9bc.png">

Reply:
Done.

Reply:
Please kindly fix this problem you have not addressed this. 

This is still your design for your mobile menu:


![Screenshot 2023-04-29 at 2 23 36 PM](https://user-images.githubusercontent.com/79506105/235302393-e154bdba-ea93-47c1-b14d-7831c06a08c7.png)